presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -326,6 +327,8 @@ else if (serverConfig.isCoordinator()) {
         binder.bind(SqlTaskManager.class).in(Scopes.SINGLETON);
         binder.bind(TaskManager.class).to(Key.get(SqlTaskManager.class));
 
+        install(new DefaultThriftCodecsModule());
```

Comment:
for Thrift codec for Airlift Unit classes, should we just fork airlift unit and add thrift related annotations there?

Reply:
I can pursue this as a follow up.